language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/ExtractToConstantCodeAction.java

```diff
@@ -128,14 +135,29 @@ private String getLocalVarName(CodeActionContext context) {
     private Position getPosition(CodeActionContext context) {
         ModulePartNode modulePartNode = context.currentSyntaxTree().get().rootNode();
         NodeList<ImportDeclarationNode> importsList = modulePartNode.imports();
-        
+
         if (importsList.isEmpty()) {
             return PositionUtil.toPosition(modulePartNode.lineRange().startLine());
         }
         ImportDeclarationNode lastImport = importsList.get(importsList.size() - 1);
         return new Position(lastImport.lineRange().endLine().line() + 2, 0);
     }
 
+    private void addRenamePopup(CodeActionContext context, CodeAction codeAction,
+                                int constDeclStrLength, int nodeStartOffset) {
+        Optional<SyntaxTree> syntaxTree = context.currentSyntaxTree();
+        if (syntaxTree.isEmpty()) {
+            return;
+        }
```

Comment:
Why do we have this check here?

Reply:
Removed this in, both ExtractToConstant and ExtractToLocalVariables code actions.